### PR TITLE
feat(ui): redesign asset & rate limit alerts

### PR DESCRIPTION
**Pull request by mavarius**
- Update clockface to 2.3.0
- Update Asset Limit alerts to be in-list
- Replace the upgrade button with rate limit alert when cardinality is exceeded
- Fix tests related to https://github.com/influxdata/clockface/issues/507

Rate Limit Alert
image

Asset Limit Alert
image

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

**Comment by mavarius** [comment]
@bpalani fyi

File: ui/src/shared/components/cloud/CloudOnly.scss [hunk 1]
```diff
@@ -1,20 +1,67 @@
...
-    padding-bottom: 16px;
+.load-data--asset-alert {
+    margin-bottom: $cf-marg-d;
 }
 
-.load-data--rate-alert {
-    padding: 0 $page-gutter;
-    padding-bottom: 16px;
-    max-width: 1608px;
+.cf-page-control-bar {
```
Comment (ischolten) [inline ui/src/shared/components/cloud/CloudOnly.scss:5]: unrelated to code: why are changes sometimes highlighted?

File: ui/src/cloud/components/RateLimitAlert.tsx [hunk 2]
```diff
@@ -37,65 +39,58 @@ interface OwnProps {
...
-      if (resource === 'cardinality') {
-        return 'total series'
-      }
-
-      return resource
-    })
-
-    return renamedResources.join(' and ')
-  }
+  return null
```
[the remark this answers is not in the document]
Comment (mavarius) [reply ui/src/cloud/components/RateLimitAlert.tsx:93]: This returns `null` if not CLOUD

File: ui/src/shared/components/cloud/CloudOnly.scss (hunk 1 above)
Comment (mavarius) [reply ui/src/shared/components/cloud/CloudOnly.scss:5]: It usually highlights it if the entire line/block didn't change but some parts of it did.
